tests/providers/class.two-factor-fido-u2f.php

```diff
@@ -1,198 +0,0 @@
-<?php
```

Comment:
Is this being renamed? Couldn't find this in a quick glance.

Reply:
class.two-factor-fido-u2f.php --> class-two-factor-fido-u2f.php